Reference/Cache/index.md

```diff
@@ -61,12 +62,14 @@ When an `ICacheRefresher` is executed via the `DistributedCache.Instance` a noti
 
 ## Events handling to refresh cache
 
-TODO: fill in these docs(https://github.com/umbraco/UmbracoDocs/issues/208)
+To use the extensions add a using to `Umbraco.Web.Cache`;  You can then call them on the DistributedCache.Instance object.
 
 ## IServerMessenger
 
-TODO: fill in these docs(https://github.com/umbraco/UmbracoDocs/issues/208)
+The server messenger broadcasts distributed cache notifications to all servers of a load balanced environment.
+Also ensures that the notification is processed on the local environment.
```

Comment:
Would it read better without starting with  Also ? eg
The server messenger ensures that the notification...